server/src/server.js

```diff
@@ -813,7 +813,7 @@ app.post("/api/set-title/:id/:domain", function(req, res) {
     return;
   }
   Shot.get(req.backend, shotId, req.deviceId, req.accountId).then((shot) => {
-    if (!shot) {
+    if (!shot.isOwner) {
```

Comment:
If I'm reading this code correctly, when `Shot.get()` is called, we don't know for sure if the shot actually exists--the `shotId` may be incorrect. If the shot isn't found, the returned `shot` will be null, so trying to access `shot.isOwner` will throw.